dotnetv3/Glacier/README.md

```diff
@@ -52,12 +51,12 @@ Or you can execute the example from within your IDE.
   configured as described in the [AWS Tools and SDKs Shared Configuration and
   Credentials Reference Guide](https://docs.aws.amazon.com/credref/latest/refdocs/creds-config-files.html).
 * .NET Core 5.0 or later
-* AWS SDK for .NET 3.0 or later
+* AWS SDK for .NET (v3) or later
 
 ## Additional resources
 * [Amazon S3 Glacier Developer Guide](https://docs.aws.amazon.com/amazonglacier/latest/dev/index.html)
 * [AWS SDK for .NET Developer Guide](https://docs.aws.amazon.com/sdk-for-net/v3/developer-guide/welcome.html)
-* [AWS SDK for .NET API Reference Guide](https://docs.aws.amazon.com/sdkfornet/v3/apidocs/index.html)
+* [AWS SDK for .NET API Reference](https://docs.aws.amazon.com/sdkfornet/v3/apidocs/index.html)
```

Comment:
Link to the specific service in the SDK reference (I think namespace makes the most sense for .NET), so: https://docs.aws.amazon.com/sdkfornet/v3/apidocs/items/Glacier/NGlacier.html

Reply:
Updated.